openbas-api/src/test/java/io/openbas/rest/InjectApiTest.java

```diff
@@ -339,6 +370,142 @@ void updateInjectForSimulationTest() throws Exception {
     assertEquals(injectTitle, JsonPath.read(response, "$.inject_title"));
   }
 
+  @DisplayName("Execute an email inject for exercise")
+  @Test
+  @WithMockPlannerUser
+  void executeEmailInjectForExerciseTest() throws Exception {
```

Comment:
to ensure a complete test, is it possible to verify the recipient and the subject of the email that has been sent?